tests/test-functions.R

```diff
@@ -0,0 +1,14 @@
+library(testit)
```

Comment:
at some point we should probably move tests to testthat. When I started working on vpc testthat wasn't so well developed yet.

Reply:
I also prefer testthat.  I'll create an issue to remind myself that change is desired.

Reply:
Ref #79